src/source/PeerConnection/PeerConnection.c

```diff
@@ -921,6 +921,20 @@ STATUS peerConnectionGetCurrentLocalDescription(PRtcPeerConnection pRtcPeerConne
     return retStatus;
 }
 
+/// @brief parses string of form "$number $whatever" returns $number as uint32
```

Comment:
Our codebase uses (or attempts to use) this commenting format:

```
/**
 * ....
 */
```

Reply:
kk